iocore/cache/Makefile.am

```diff
@@ -27,7 +27,8 @@ AM_CPPFLAGS += \
 	-I$(abs_top_srcdir)/proxy/http/remap \
 	-I$(abs_top_srcdir)/mgmt \
 	-I$(abs_top_srcdir)/mgmt/utils \
-	$(TS_INCLUDES)
+	$(TS_INCLUDES) \
+	@YAMLCPP_INCLUDES@
```

Comment:
This is needed unfortunately.
```
In file included from Cache.cc:34:
In file included from /Users/mkitajo/src/github.com/trafficserver/proxy/http/HttpSM.h:40:
In file included from /Users/mkitajo/src/github.com/trafficserver/proxy/http/HttpTransact.h:28:
In file included from /Users/mkitajo/src/github.com/trafficserver/iocore/hostdb/P_HostDB.h:36:
In file included from /Users/mkitajo/src/github.com/trafficserver/iocore/dns/P_SplitDNS.h:33:
In file included from /Users/mkitajo/src/github.com/trafficserver/iocore/dns/P_DNS.h:36:
In file included from /Users/mkitajo/src/github.com/trafficserver/iocore/net/P_Net.h:115:
In file included from /Users/mkitajo/src/github.com/trafficserver/iocore/net/P_QUICNetVConnection.h:53:
In file included from /Users/mkitajo/src/github.com/trafficserver/iocore/net/quic/QUICFrameDispatcher.h:31:
In file included from /Users/mkitajo/src/github.com/trafficserver/iocore/net/quic/QUICContext.h:28:
/Users/mkitajo/src/github.com/trafficserver/iocore/net/quic/QUICLog.h:6:10: fatal error: 'yaml-cpp/yaml.h' file not found
```

Reply:
I pushed a commit (a788cc38ca803a0f3c5901b14db5f6a7ce6361e4 ) to remove the dependency. Now we should be able to build without adding `YAMLCPP_INCLUDES` to other modules.

Reply:
will rebase it